start.py

```diff
@@ -25,7 +25,7 @@ def main():
 
     env_builder = venv.EnvBuilder(with_pip=True)
     venv_context = env_builder.ensure_directories(venv_path)
-    python_run(venv_context, "autorest.jsonrpc.server", *sys.argv[1:])
+    python_run(venv_context, ["autorest", "jsonrpc", "server"], *sys.argv[1:])
```

Comment:
Was that change expected?

Reply:
yeah, because we changed the command parameter of python run to take in a list,  instead of splitting on the space

Reply:
yeah but here it was a single argument (not separated with space)

Reply:
oh wait, you're right, this is the module, not the command